ledger-service/http-json/src/main/scala/com/digitalasset/http/Endpoints.scala

```diff
@@ -756,9 +774,9 @@ class Endpoints(
       case o => o
     }
 
-  private def httpResponse[A: JsonWriter](
-      result: ET[domain.SyncResponse[A]]
-  )(implicit metrics: Metrics): Future[HttpResponse] = {
+  private implicit def hrFullySync[A: JsonWriter](implicit
```

Comment:
What does `hr` stand for?

Reply:
Re-reading now I guess `httpResponse`, but it could be nice to be expanded for other readers.

Reply:
I agree here, `hr` is a bit short.

Reply:
ab815576341bed342834e4765a2dd4c030543c2d